src/qos_client/src/yubikey.rs

```diff
@@ -350,21 +351,35 @@ fn generate_random_rfc5280_serial() -> [u8; 20] {
 	serial
 }
 
-#[test]
-fn test_rfc5280_serial_generation_success() {
-	use x509_cert::certificate::Rfc5280;
+#[cfg(test)]
+mod tests {
```

Comment:
The compiler spends more time eliminating dead code that is not gated behind a `cfg` flag.